src/pages/HomePage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const HomePage = () => {
+  return (
+    <div>
+      <h2>Welcome to SecondhandMovies.com</h2>

```

Comment:
On the Slack channel we came up with either Caaakewalk DVDs or Re-run DVD's for website name

Reply:
Caaakewalk DVDs gets my vote